<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoTile</class>
 <widget class="QWidget" name="VideoTile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>162</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>288</width>
    <height>162</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>288</width>
    <height>162</height>
   </size>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel, #author {
	color: rgba(255,255,255,190);
}

#lenght {
	font: 20px;
	font: bold;
}

QPushButton {
    border: none;
}
#playButton:!hover {
    background-color: rgba( 0,0,0, 100);
}
#playButton:hover:!pressed {
    background-color:   rgba( 0, 0, 0, 120 );
}
#playButton:pressed {
    background-color:   rgba( 0, 0, 0, 140);
}

#infoOverlay {
	background-color: rgba(0,0,0,120);
}

#title {
	font: 16pt
}

#titleSep {
	color: rgba(255,255,255,100)
}

#author {
	font: 16px;
}

#by {
	font: 9px
}

#uploaded {
	font: 16px;
}

#dislikeBar, #likeBar {
	border: none;
	background-color: rgba(0,0,0,0);
}

#dislikeBar::chunk {
   background-color: rgba(255,0,0,70);
}

#likeBar::chunk {
	background-color: rgba(0,255,0,70);
}

#ratingOverlay {
	background-color: rgba(0,0,0,70);
}
#likeCount {
	color: rgb(50,255,50);
}

#dislikeCount {
	color: rgb(255,70,30);
}

#viewCount {
	color: rgb(30, 120, 255);
}


</string>
  </property>
  <widget class="QLabel" name="image">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>288</width>
     <height>162</height>
    </rect>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>288</width>
     <height>162</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <widget class="QWidget" name="overlaylayout" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>288</width>
      <height>162</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QWidget" name="infoOverlay" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="title">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>80</height>
           </size>
          </property>
          <property name="text">
           <string>Title</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignHCenter|Qt::AlignTop</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="titleSep">
          <property name="frameShape">
           <enum>QFrame::HLine</enum>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>25</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="by">
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>By: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="author">
             <property name="cursor">
              <cursorShape>PointingHandCursor</cursorShape>
             </property>
             <property name="text">
              <string>author</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>8</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="uploaded">
             <property name="text">
              <string>uploaded</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="ratingBar" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>6</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>6</height>
        </size>
       </property>
       <widget class="QProgressBar" name="dislikeBar">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>6</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="invertedAppearance">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QProgressBar" name="likeBar">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>100</width>
          <height>6</height>
         </rect>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="lenght">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>139</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>00:00:00</string>
    </property>
   </widget>
   <widget class="QWidget" name="ratingOverlay" native="true">
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>112</y>
      <width>41</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="viewCount">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="likeCount">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dislikeCount">
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
